tools/interop_matrix/client_matrix.py

```diff
@@ -35,14 +35,15 @@ def get_release_tags(lang):
 
 def get_runtimes_for_lang_release(lang, release):
     """Get list of valid runtimes for given release of lang."""
-    runtimes_to_skip = []
-    release_info = LANG_RELEASE_MATRIX[lang][release]
-    if release_info:
-        runtimes_to_skip = release_info.skip_runtime
-    return [
-        runtime for runtime in LANG_RUNTIME_MATRIX[lang]
-        if runtime not in runtimes_to_skip
-    ]
+    runtimes = list(LANG_RUNTIME_MATRIX[lang])
+    release_info = LANG_RELEASE_MATRIX[lang].get(release)
+    if release_info and release_info.runtime_subset:
```

Comment:
Minor point: you could avoid a little unnecessary validation work when runtime_subset is not set.  Something like:

```py
def get_runtimes_for_lang_release(lang, release):
    """Get list of valid runtimes for given release of lang."""
   release_info = LANG_RELEASE_MATRIX[lang].get(release)
   if not release_info or not release_info.runtime_subset:
       return list(LANG_RUNTIME_MATRIX[lang])

   runtimes = list(release_info.runtime_subset)
   # check that ....
```

Reply:
makes sense, but I think the complexity of what you're proposing is about the same as what I currently have, so I'll leave as to avoid a second round of testing.